.github/workflows/trigger-breaking-change-alert.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
   trigger-notifier:
     if: contains(github.event.pull_request.labels.*.name, 'breaking')
     secrets: inherit
-    uses: rapidsai/shared-workflows/.github/workflows/breaking-change-alert.yaml@branch-25.02
+    uses: rapidsai/shared-workflows/.github/workflows/breaking-change-alert.yaml@cuda-12.8.0
```

Comment:
Just making sure, the plan is the usual of merging this PR as-is and then reverting to 25.02 once everything is merged without deleting the branch, yes?

Reply:
Yes, that’s correct.